exercises/java/src/main/java/com/adaptionsoft/games/uglytrivia/Game.java

```diff
@@ -2,27 +2,26 @@
 
 import java.util.ArrayList;
 import java.util.LinkedList;
+import java.util.List;
 
 public class Game {
-    ArrayList players = new ArrayList();
-    int[] places = new int[6];
-    int[] purses  = new int[6];
-    boolean[] inPenaltyBox  = new boolean[6];
+	public static final int NUMBER_OF_COINS_TO_WIN = 6;
+	List<Player> players = new ArrayList<>();
     
     LinkedList popQuestions = new LinkedList();
     LinkedList scienceQuestions = new LinkedList();
     LinkedList sportsQuestions = new LinkedList();
     LinkedList rockQuestions = new LinkedList();
-    
-    int currentPlayer = 0;
+
+    int currentPlayerIndex = 0;
     boolean isGettingOutOfPenaltyBox;
     
     public  Game(){
-    	for (int i = 0; i < 50; i++) {
-			popQuestions.addLast("Pop Question " + i);
-			scienceQuestions.addLast(("Science Question " + i));
-			sportsQuestions.addLast(("Sports Question " + i));
-			rockQuestions.addLast(createRockQuestion(i));
+    	for (int count = 0; count < 50; count++) {
+			popQuestions.addLast("Pop Question " + count);
+			scienceQuestions.addLast(("Science Question " + count));
+			sportsQuestions.addLast(("Sports Question " + count));
+			rockQuestions.addLast(createRockQuestion(count));

```

Comment:
Nice work on that loop, it seems like there could be even more done to make it a bit more consistent like either using methods for or categories or getting rid of the createRockQuestion method.

Reply:
I agree I will remove the rock method